### DISCUSSION TITLE
Transport in the network: Implicit, includes transport means, cooling, losses, taxes and tariffs

### DISCUSSION THREAD

**Discussion by tngTUDOR** [Depart-de-Sentier/sentier.dev#18] [General]
+ This must be implemented as an `optional` feature when building the inventory for a given foreground/functional unit.

We could implement a "TransportInferenceFunction" that could work in 3 different "tiers":

## T1
 Using fixed average statistics (like a table that specifies average transport means per product type table 4.2 of EI guidelines report)
## T2
given an amount of an input, we identify from COMTRADE, based on the density and the value of the produc,t what is the average `distance of transport`. With this, we can infer if it is domestically sourced, or if it is the object of international trade. 
When domestically sourced, we know the distance + volume + density will dictate the transport mode.
When _internationally_ traded, we need COMTRADE or BACI to find out "average in a given year" who were the mayor trading partners supplying the product. We have the volumes, and the distance,  transport mode will be inferred for each trading partner (weighted by their trade volume)

## T3
Using specific locations + Open data about distances (open street map) the distance can be calculated, and used to determine the actual transport means.

**Comment by michaelweinold** [comment]
Tier 3:
<img width="500" alt="Screenshot 2024-04-13 at 16 36 31" src="https://github.com/Depart-de-Sentier/sentier.dev/assets/23102087/e594a75a-354f-4358-99ae-b398cb5c4d17">
Tier 2:
<img width="500" alt="Screenshot 2024-04-13 at 16 36 58" src="https://github.com/Depart-de-Sentier/sentier.dev/assets/23102087/a1d01557-d070-4689-86dd-f1a83806945e">
Tier 1:
<img width="500" alt="Screenshot 2024-04-13 at 16 37 08" src="https://github.com/Depart-de-Sentier/sentier.dev/assets/23102087/5437af84-2cac-426c-8a32-48a0a827e969">

**Comment by mklarmann** [comment]
Use industrial centers of productions from:
-    openstreetmap could provide tags, and we could calculate the density here
-    For this... "open supply hub" for industrial facilities.

determining freezing/cooling based on a step function on cooling.

**Reply by jsvgoncalves** [reply]
https://opensupplyhub.org/

**Comment by cmutel** [comment]
@tngTUDOR  says that https://opensupplyhub.org/ has information on locations of large producers.

**Comment by mklarmann** [comment]
The opensupplyhub database contains geolocations for 300k production facilities across various sectors. The OS ID used to identify the facilities can be used.
